cmd/libsnap-confine-private/snap.c

```diff
@@ -48,6 +48,34 @@ bool verify_security_tag(const char *security_tag)
 	return (status == 0);
 }
 
+bool verify_hook_security_tag_name(const char *security_tag)
```

Comment:
We have a NACK on using regular expressions. This needs to be re-written using other logic.